### PR TITLE
Add tests for v2/views.py

### PR DESCRIPTION
**Pull request by richardzilincikPantheon**

### CODE REVIEW

**Comment by richardzilincikPantheon** [comment]
This incorporates changes from #65 and should not be merged before it

**Comment by SlavomirMazurPantheon** [comment]
Please use real RFC and draft documents. You can place them to some directory like tests/resources/

File: tests/resources/settings.py [hunk 1]
```diff
@@ -0,0 +1,198 @@
...
+Generated by 'django-admin startproject' using Django 2.1.2.
+
+For more information on this file, see
+https://docs.djangoproject.com/en/2.1/topics/settings/
+
+For the full list of settings and their values, see
+https://docs.djangoproject.com/en/2.1/ref/settings/
+"""
+
+__author__ = "Miroslav Kovac, Carl Moberg"
```
Comment (miroslavKovacPantheon) [inline tests/resources/settings.py:28]: with new files please get us out as authors and put your name only same with email. and copyright must have current year no Cisco anymore this is only IETF project. we are adding IETF to files were cisco was but new files contain only IETF

File: tests/resources/settings.py [hunk 2]
```diff
@@ -0,0 +1,198 @@
+# Copyright The IETF Trust 2019, All Rights Reserved
```
Comment (miroslavKovacPantheon) [inline tests/resources/settings.py:1]: same here No Cisco only IETF with correct date

File: tests/resources/settings.py [hunk 3]
```diff
@@ -0,0 +1,198 @@
...
+    },
+]
+
+WSGI_APPLICATION = 'yangvalidator.wsgi.application'
+
+
+# Database
+# https://docs.djangoproject.com/en/2.1/ref/settings/#databases
+
+DATABASES = {
```
Comment (miroslavKovacPantheon) [inline tests/resources/settings.py:115]: do we need databases in this project at all? as far as I remeber not.. if not please remove this section completely

File: yangvalidator/settings.py [hunk 4]
```diff
@@ -112,12 +112,12 @@
 # Database
 # https://docs.djangoproject.com/en/2.1/ref/settings/#databases
 
-#DATABASES = {
-#    'default': {
-#        'ENGINE': 'django.db.backends.sqlite3',
-#        'NAME': os.path.join(BASE_DIR, 'db.sqlite3'),
-#    }
-#}
+DATABASES = {
```
Comment (miroslavKovacPantheon) [inline yangvalidator/settings.py:115]: do we need databases in this project at all? as far as I remeber not.. if not please remove this section completely

File: tests/test_views.py [hunk 5]
```diff
@@ -0,0 +1,383 @@
+import json
```
Comment (miroslavKovacPantheon) [inline tests/test_views.py:1]: missing copyright and author

**Review by miroslavKovacPantheon — CHANGES_REQUESTED** [review]

File: tests/resources/settings.py (hunk 1 above)
Comment (richardzilincikPantheon) [reply tests/resources/settings.py:28]: Most of the file was copied over and unchanged, so I think the cisco copyright notice should be kept.

**Pull request merged**